client/src/components/Sidebar/index.tsx

```diff
@@ -1,29 +1,21 @@
 import React from "react";
+import classNames from "classnames";
 import { SidebarButton } from "@components/core";
 import useSidebar from "@components/Sidebar/useSidebar";
+import styles from "./styles.module.scss";
 
 const Sidebar = () => {
 	const { isHidden, toggleSidebar } = useSidebar();
 
+	const sidebarClass = classNames(styles.sidebar, {
+		[styles.isDisplayed]: !isHidden,
+	});
+
 	return (
-		<div
-			className="Sidebar"
-			style={
-				isHidden
-					? {}
-					: {
-							background: "ghostwhite",
-							width: "20%",
-							height: "100vh",
-							textAlign: "left",
-							display: "flex",
-							flexDirection: "column",
-							justifyContent: "space-around",
-						}
-			}
-		>
```

Comment:
I think we should keep the styles in their own file, So I moved these guys to `client/src/components/Home/styles.module.scss`